docs/src/modules/components/AppFrame.js

```diff
@@ -263,6 +263,7 @@ class AppFrame extends React.Component {
                       color="inherit"
                       aria-owns={languageMenu ? 'language-menu' : undefined}
                       aria-haspopup="true"
+                      aria-label="Change language"
```

Comment:
I have fixed a couple of issues reported by https://chrome.google.com/webstore/detail/wave-evaluation-tool/jbbplnpkjmmeebjpijfedlgcdilocofh.